### DISCUSSION TITLE
how to compile mameui on vs2022 and run with mame?

### DISCUSSION THREAD

**Discussion by backrock** [mamedev/discussions#63] [General]
Hi all,
I checkout the mameui from https://github.com/Robbbert/mameui, 
1. I compiled by gcc on msys32 and run mameui.exe but exit with no message when the load progress done.
2. I generate the VS use "make SUBTARGET=tiny PTR64=1 TOOLS=1 OPTIMIZE=0 NOWERROR=1 MODERN_WIN_API=1 NO_USE_PORTAUDIO=1 vs2019 -j4" , after fix several errors then compile success, and I run mameui.exe but the UI was same as mame.exe, 
my question is how to use mameui? how can I replace the mame origin UI by mameui? 
thanks.

**Comment by backrock** [comment]
https://github.com/Robbbert/mameui
I compiled again by vs2022, 
make OSD=winui USE_SYSTEM_LIB_PORTAUDIO=1 PTR64=1 TOOLS=1 OPTIMIZE=0 NOWERROR=1 MODERN_WIN_API=1 NO_USE_PORTAUDIO=1 vs2019 -j4
but still show the old UI, who know how to display the winui?
![mameui6](https://github.com/mamedev/discussions/assets/32918508/f3cf6109-120e-4035-be99-e0e1f8f386c2)

**Comment by galibert** [comment]
We (mamedev) do not directly support the mameui derivative.  It is Robbert's work.  Please use the issues or discussions on https://github.com/Robbbert/mameui to get support with your problems.
